tests/data/validators/models/FakedValidationModel.php

```diff
@@ -88,4 +88,12 @@ public function getInlineValArgs()
     {
         return $this->inlineValArgs;
     }
+
+    /**
+     * @see https://github.com/yiisoft/yii2/issues/14370
+     */
+    private function number()
```

Comment:
This bug happen when you have function in model class with name that same to one of built-in validators. And Validator::createValidator create InlineValidator instead NumberValidator in this case. I can make separate model class and test for getting this more clear.